### PR TITLE
feat: allow passing judge server details in a secret

### PR DESCRIPTION
**Pull request by leseb**
cf5c364 feat: allow passing judge server details in a secret

commit cf5c364697f19e5d13679748dad3892fd975f611
Author: Sébastien Han <seb@redhat.com>
Date:   Wed Oct 16 17:45:44 2024 +0200

    feat: allow passing judge server details in a secret
    
    We can now use `--judge-serving-model-secret` to provide an existing
    Secret that contains details about how to connect to the judge server
    during the evaluation steps.
    
    The secret must contain the following keys:
    
    * `JUDGE_API_KEY`: The API key for the model to evaluate - **Required**
    * `JUDGE_ENDPOINT`: Serving endpoint for evaluation - **Required**
    * `JUDGE_NAME`: The name of the model to use for evaluation - **Required**
    
    Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

File: standalone/README.md [hunk 1]
```diff
@@ -47,11 +47,14 @@ The `standalone.py` script includes a main command to execute the full workflow,
...
 
 1. CLI Options or/and Environment Variables: Supply all necessary information via CLI options or environment variables.
-    * See [CLI Options](#cli-options) for full list.
-2. Kubernetes Secret: Provide the name of a Kubernetes secret that contains all relevant details using the `--sdg-object-store-secret` option.
+    * See [CLI Options](#cli-options) for full list. In particular `--sdg-object-store-*` and `--judge-serving-model-*` options.
+2. Kubernetes Secret: Provide the name of a Kubernetes secret that contains all relevant details
+   using the `--sdg-object-store-secret` option and `--judge-serving-model-secret` option.
     *  See [Creating the Kubernetes Secret for S3 Details](#creating-the-kubernetes-secret-for-s3-details) for information on how to create the secret.
 
 The examples below assume there is a secret in `my-namespace` named `sdg-data` that holds information about the S3 bucket.
```
Comment (sallyom) [inline standalone/README.md:60]: also update #L60-62, something like:

The examples below assume there are the following secrets in `my-namespace`: `sdg-data` secret that includes information about the S3 bucket and `judge-serving-details` secret that includes information about the judge server model. A judge server model is assumed to run external to the script. See [Judge Model Details](#judge-model-details) for required information.

**Review by sallyom — COMMENTED** [review]
LGTM, I haven't tested w/ it yet though! I made a suggestion to update the docs examples.

File: standalone/README.md (hunk 1 above)
Comment (leseb) [reply standalone/README.md]: Addressed thanks.

**Comment by leseb** [comment]
We need to make progress, I verified this locally. As much as I dislike doing this, I'm self-merging. Thanks!

**Pull request merged**